### ISSUE TITLE
JsonIgnore annotation seems to be broken

### ISSUE THREAD

**Issue by markrietveld** [bluelinelabs/LoganSquare#20]
This model class:
```java
import com.bluelinelabs.logansquare.annotation.JsonIgnore;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject(fieldDetectionPolicy = JsonObject.FieldDetectionPolicy.NONPRIVATE_FIELDS_AND_ACCESSORS)
public class Test {
    public String shouldBeProcessed;
    @JsonIgnore
    public String shouldNotBeProcessed;
}
```

Generates this code for me:
```java
public static void parseField(Test instance, String fieldName, JsonParser jsonParser) throws IOException {
    if ("shouldBeProcessed".equals(fieldName)) {
      instance.shouldBeProcessed = jsonParser.getValueAsString(null);
    } else if ("shouldNotBeProcessed".equals(fieldName)){
      instance.shouldNotBeProcessed = jsonParser.getValueAsString(null);
    }
  }

public static void _serialize(Test object, JsonGenerator jsonGenerator, boolean writeStartAndEnd) throws IOException {
    if (writeStartAndEnd) {
      jsonGenerator.writeStartObject();
    }
    jsonGenerator.writeStringField("shouldBeProcessed", object.shouldBeProcessed);
    jsonGenerator.writeStringField("shouldNotBeProcessed", object.shouldNotBeProcessed);
    if (writeStartAndEnd) {
      jsonGenerator.writeEndObject();
    }
  }
```

**Comment by EricKuck** [comment]
Thanks for the report. Will be fixed very soon.

**Comment by EricKuck** [comment]
This is fixed in commit 0209e29159d91c2a4c6d6a55eab9c6c63426b4fb. Will be getting uploaded to JCenter and Maven Central as version 1.0.4 before the end of the day.

**Comment by EricKuck** [comment]
1.0.4 has been pushed to JCenter and Maven Central, so you should be good to go now. Sorry about the issue!
